ambry-clustermap/src/main/java/com.github.ambry.clustermap/FixedBackoffResourceStatePolicy.java

```diff
@@ -88,11 +89,9 @@ public boolean isDown() {
         down = true;
       }
     }
-    logger.trace("Resource " + resource + " is " + (down ? "down" : "not down")
-        + "; failureCount: " + failureCount.get()
-        + "; failureCountThreshold: " + failureCountThreshold
-        + "; remaining time: " + (downUntil.get() - SystemTime.getInstance().milliseconds())
-        + "; hard down: " + hardDown);
+    logger.trace("Resource {} is {}; failureCount: {}; failureCountThreshold: {}; remaining time: {}; hard down: {}",
+        resource, (down ? "down" : "not down"), failureCount.get(), failureCountThreshold,
+        (downUntil.get() - SystemTime.getInstance().milliseconds()), hardDown);
```

Comment:
remaining time only makes sense if the state is down. In the up case `downUntil` could be in the past and this could print a negative, misleading value.